tests/middleware/test_message_state.py

```diff
@@ -88,7 +90,7 @@ def wait():
         assert type(data) == result_type
 
     @pytest.mark.parametrize(
-        "max_size,expected", [pytest.param(0, []), pytest.param(1000, [5])],
+        "max_size,expected", [pytest.param(0, "[]"), pytest.param(100, "(5,)")],
```

Comment:
For the tests in these files could you test all the fields of the state ? For example started datetime and such. To do tests using datetime.now you can install https://github.com/spulec/freezegun as a test dependency. It make all call to datetime.now return the same value (make it easy to test).

Reply:
cool, thanks @antoinerabany